pytorch_lightning/trainer/training_loop.py

```diff
@@ -606,7 +606,7 @@ def optimizer_closure():
                 self.get_model().on_batch_end()
 
         # update progress bar
-        if batch_idx % self.progress_bar_refresh_rate == 0:
+        if self.show_progress_bar and batch_idx % self.progress_bar_refresh_rate == 0:
```

Comment:
is the `show_progress_bar` needed here?

Reply:
I'll change it to not use the `show_progress_bar`, but if refresh rate is 0 I'm pretty sure Python throws a divide by 0 error

Reply:
`if self.progress_bar_refresh_rate and batch_idx % self.progress_bar_refresh_rate == 0:`

Reply:
Cleaner than mine :+1: